ci/tasks/promote-candidate.sh

```diff
@@ -40,8 +40,8 @@ EOF
   git diff | cat
   git add .
 
-  git config --global user.email wangjq@cn.ibm.com
-  git config --global user.name wangjq
+  git config --global user.email zhanggbj@cn.ibm.com
+  git config --global user.name zhanggbj
```

Comment:
Do we really need your github ID here? 

I am merging to move things but if there is a way to avoid then let's do so. Best.